### PR TITLE
Containerapp 0.3.13 Release

### PR DESCRIPTION
**Pull request by StrawnSC**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Containerapp

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py [hunk 1]
```diff
@@ -47,6 +47,77 @@ def test_containerapp_env_e2e(self, resource_group):
             JMESPathCheck('length(@)', 0),
         ])
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="australiaeast")
+    @StorageAccountPreparer(location="australiaeast")
+    @live_only()  # encounters 'CannotOverwriteExistingCassetteException' only when run from recording (passes when run live)
```
Comment (zhoxing-ms) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py:53]: May I ask what is the reason for this problem `'CannotOverwriteExistingCassetteException' only when run from recording`?

Comment (StrawnSC) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py:53]: If I recall correctly, this test actually passes locally (when run from a recording or live), but fails when run in the CI. Let me try rerecording again to see if it works

Comment (StrawnSC) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py]: Yes, I just double-checked this with a rerecorded test. The test passes locally, but fails the CI for some reason. I think the best thing to do here is mark it as live_only and investigate further when we have time

Comment (zhoxing-ms) [reply src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py]: Ok, please remember to submit PR separately to solve the `@live_only()` problem in the future. It is recommended that you create a work item for these testing problems

**Comment by zhoxing-ms** [comment]
> AssertionError: Can't overwrite existing cassette ('/mnt/vss/_work/1/s/src/containerapp/azext_containerapp/tests/latest/recordings/test_containerapp_env_logs_e2e.yaml') in your current record mode ('once').
No match for the request (<Request (PUT) [https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.App/managedEnvironments/containerapp-env000003/providers/Microsoft.Insights/diagnosticSettings/diagnosticsettings?api-version=2017-05-01-preview>](https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.App/managedEnvironments/containerapp-env000003/providers/Microsoft.Insights/diagnosticSettings/diagnosticsettings?api-version=2017-05-01-preview%3E)) was found.
Found 2 similar requests with 1 different matcher(s) :

> 1 - (<Request (PUT) [https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.App/managedEnvironments/containerapp-env000003/providers/Microsoft.Insights/diagnosticSettings/diagnosticsettings?api-version=2021-05-01-preview>).](https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.App/managedEnvironments/containerapp-env000003/providers/Microsoft.Insights/diagnosticSettings/diagnosticsettings?api-version=2021-05-01-preview%3E).)
Matchers succeeded : ['method', 'scheme', 'host', 'port', 'path']
Matchers failed :
_custom_request_query_matcher - assertion failure :
None

@StrawnSC This is caused by the inconsistent api-version of `diagnostic_settings` (introduced by PR #23569). It is recommended that you pull the latest CLI code from main CLI repo and then try to re-record the tests

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**